ui/desktop/app/templates/scopes/scope/projects/sessions/index.hbs

```diff
@@ -173,9 +172,6 @@
                   {{format-date-iso-human model.created_time}}
                 </time>
               </row.cell>
-              <row.cell>
-                <SessionStatus @model={{model}} />
-              </row.cell>
```

Comment:
Did we want to remove this? I thought we were just removing the active label for targets

Reply:
@ZedLi @randallmorey I think this was the main issue. The status badge doesn't update if the user terminates the session, via the user's terminal, only when the `cancel` action is pressed.

Reply:
I believe that's expected since killing the SSH connection doesn't end the session. See this [thread](https://hashicorp.slack.com/archives/C012GTH1C4E/p1686693010861389). 

Theoretically we could fix the active badge on the target issue by refreshing both sessions and targets when manually refreshing so the sessions gets updated as well which would then correctly know if its active, but it probably makes more sense to remove it. 

I would assume we want to keep the session status though.

Reply:
@ZedLi thank you for the information let me work through a quick refactor